tests/unittests/test_handler/test_handler_reset_rmc.py

```diff
@@ -0,0 +1,44 @@
+from cloudinit.config import cc_reset_rmc
+
+from cloudinit import cloud
+from cloudinit import helpers
+from cloudinit import util
+
+from cloudinit.tests import helpers as t_help
+
+import logging
+import os
+
+LOG = logging.getLogger(__name__)
+
+
+class TestRsctNodeFile(t_help.FilesystemMockingTestCase):
+    def setUp(self):
+        super(TestRsctNodeFile, self).setUp()
+        self.tmp = self.tmp_dir()
+
+    def cc_cloud(self):
+        paths = helpers.Paths({})
+        ds = None
+        return cloud.Cloud(ds, paths, {}, None, None)
+
+    def rsct_node(self):
+        file_content = '23edfgrkahidksla344'
+        node_file = '/etc/ct_node_id'
+        if os.path.exists(node_file):
+            os.remove(node_file)
+        with open(node_file) as fp:
+            fp.write(file_content)
+        cc = self.cc_cloud()
+        cc_reset_rmc.handle('test', None, cc, LOG, [])
+        contents = util.load_file(node_file)
+        self.assertNotEqual(file_content, contents)
+
+    def firewall_status_check(self):
+        firewall_status = 'systemctl is-active --quiet firewalld'
+        restart_status = os.system(firewall_status)
```

Comment:
this wont work everywhere, and we don't want to depend on state of the host system for the tests.